docker/Dockerfile

```diff
@@ -1,18 +1,19 @@
 FROM tensorflow/tensorflow:1.7.0-py3
-
+ARG FLOWDEC_REPO="https://github.com/hammerlab/flowdec.git"
+ARG CODEX_REPO="-b r0.1 https://github.com/eric-czech/codex.git"
```

Comment:
`s/eric-czech/hammerlab/g`?

Reply:
good call, I'll probably forget to change that if I don't do it now -- thanks!